qa/common/infer_util.py

```diff
@@ -702,6 +702,7 @@ def inferAndCheckResults(
                                 if cidx == expected0_sort_idx[b][idx]:
                                     tester.assertEqual(
                                         ctuple[2],
+                                        ctuple[2].strip("\r"),
                                         "label{}".format(expected0_sort_idx[b][idx]),
```

Comment:
`assertEqual(a, b, msg=None)`: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertEqual


Should this be replacing line 704 instead of adding a new line on 705?
```suggestion
                                    tester.assertEqual(
                                        ctuple[2].strip("\r"),
                                        "label{}".format(expected0_sort_idx[b][idx]),
```

Reply:
Yes, good catch! I was updating this based on a text file of a patch and clearly blurred things together.